server/core/src/miscServerFunct.cpp

```diff
@@ -2023,8 +2032,18 @@ longNoSupport( ... ) {
     return ( rodsLong_t ) SYS_NOT_SUPPORTED;
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
 int
 svrPortalPutGetRbudp( rsComm_t *rsComm ) {
+#pragma clang diagnostic pop
+
+    const irods::at_scope_exit add_deprecated_api_warning_message{[&] {
+        addRErrorMsg(&rsComm->rError, DEPRECATED_API, "RBUDP has been deprecated.");
+    }};
+    
+    irods::experimental::log::server::warn("Warning: RBUDP transfer option was called.  This has been deprecated.");
```

Comment:
I'm not sure that we need to log a message for this. The rError message should be enough.

Reply:
That being said, I'm not opposed to it. I defer to others.

Reply:
Done